controllers:
  joint1_controller:
    type: JointAngVelAccController
    joint_name: joint1
    max_ang: 1.57
    min_ang: -1.57
    max_vel: 1.0
    max_acc: 1.0

  joint2_controller:
    type: JointAngVelAccController
    joint_name: joint2
    max_ang: 1.57
    min_ang: -1.57
    max_vel: 1.0
    max_acc: 1.0

  joint3_controller:
    type: JointAngVelAccController
    joint_name: joint3
    max_ang: 1.57
    min_ang: -1.57
    max_vel: 1.0
    max_acc: 1.0

  joint4_controller:
    type: JointAngVelAccController
    joint_name: joint4
    max_ang: 1.57
    min_ang: -1.57
    max_vel: 1.0
    max_acc: 1.0

  joint5_controller:
    type: JointAngVelAccController
    joint_name: joint5
    max_ang: 1.57
    min_ang: -1.57
    max_vel: 1.0
    max_acc: 1.0

  arm_controller:
    type: JointTrajectoryActionController
    joint_trajectory_action_node:
      min_velocity: 0.1
      constraints:
        goal_time: 0.05
    joint_controllers:
      - "joint1_controller"
      - "joint2_controller"
      - "joint3_controller"
      - "joint4_controller"
      - "joint5_controller"

#  gripper_controller:
#    type: JointTrajectoryActionController
#    joint_trajectory_action_node:
#      min_velocity: 0.1
#      constraints:
#        goal_time: 0.05
#    joint_controllers:
#      - "r_joint_controller"